src/openrct2/paint/track/coaster/TwisterRollerCoaster.cpp

```diff
@@ -24,27 +24,26 @@
 #include "../../track/Support.h"
 
 using namespace OpenRCT2;
-
```

Comment:
I'd prefer there to always be an empty line after `using namespace ...;`.

Reply:
I’ll include it in the next refactor, which I have already prepared.